apps/condo/pages/employee/index.tsx

```diff
@@ -133,9 +134,12 @@ export const EmployeesPageContent = ({
 }
 
 const EmployeesPage = () => {
-    const { link: { role = {} }, organization }  = useOrganization()
+    const { link, organization }  = useOrganization()
     const userOrganizationId = get(organization, 'id', null)
-    const canManageEmployee = get(role, 'canInviteNewOrganizationEmployees', null)
+    const canManageEmployee = get(link, 'role.canInviteNewOrganizationEmployees', null)
+    const employeeId = get(link, 'id')
+
+    usePreviousQueryParams({ trackedParamNames: ['sort', 'filters'], employeeId })
```

Comment:
Behavior with saving filters is needed on all pages with filters?

Mb move it to some `Context`, use it in `_app` and not not copy paste this logic everywhere where we need filters?